src/backend/joanie/core/models/products.py

```diff
@@ -465,33 +465,34 @@ class OrderGroup(BaseModel):
     is_active = models.BooleanField(_("is active"), default=True)
     position = models.PositiveSmallIntegerField(
         _("priority"),
-        help_text=_("Priority of the order group"),
+        help_text=_("Priority"),
         default=None,
         null=True,
         blank=True,
     )
     start = models.DateTimeField(
-        help_text=_("Date at which the order group activation begins"),
-        verbose_name=_("order group start datetime"),
+        help_text=_("Date at which the order group rule starts"),
+        verbose_name=_("rule starts at"),
         blank=True,
         null=True,
     )
     end = models.DateTimeField(
-        help_text=_("Date at which the order group activation ends"),
-        verbose_name=_("order group end datetime"),
+        help_text=_("Date at which the order group rule ends"),
+        verbose_name=_("rule ends at"),
         blank=True,
         null=True,
     )
     discount = models.ForeignKey(
         to="Discount",
-        verbose_name=_("discount on the product price for the order group"),
+        verbose_name=_("Product price discount"),
         related_name="order_groups",
         on_delete=models.CASCADE,
         null=True,
         blank=True,
     )
 
     class Meta:
+        ordering = ["course_product_relation", "position"]
```

Comment:
is it too late to add the informations we normally add in a model ?
```suggestion
        db_table = "joanie_order_group"
        verbose_name = _("order group")
        verbose_name_plural = _("order groups")
        ordering = ["course_product_relation", "position"]
```